tailwind.config.js

```diff
@@ -7,6 +7,11 @@ const reuseColors = {
   "orange-100": "#FB8F0A",
 };
 
+const reuseComponentsLengths = {
+  "header-desktop": "130px",
```

Comment:
The unit should be consistent with Tailwind. Tailwind uses ``rem`` by default.

Reply:
Fixed on [`97b4954` (#15)](https://github.com/LauraBeatris/floripamais/pull/15/commits/97b4954955eac66013b8605995660731f7c69dff)